telegram/ext/filters.py

```diff
@@ -2075,6 +2075,18 @@ def filter(self, message: Message) -> bool:
                 and any(message.from_user.language_code.startswith(x) for x in self.lang)
             )
 
+    class _Attachment(MessageFilter):
+        name = 'Filters.attachment'
+
+        def filter(self, message: Message) -> bool:
+            return bool(message.effective_attachment)
+
+    attachment = _Attachment()
+    """Messages where :meth:`telegram.Message.effective_attachment` is not None.
```

Comment:
```suggestion
    """Messages that contain :meth:`telegram.Message.effective_attachment`.
```
? This matches the other docstrings. or at least ``:obj:`None` `` 😉